plugins/platform-ui/src/components/internal/Root.svelte

```diff
@@ -53,7 +53,7 @@
         currentAppErr = `There is no application route defined for ${appRoute}`
       }
     },
-    { application: '#default' }
+    { application: platform.getMetadata(routeMeta(defaultApp))?.route || '#default' }
```

Comment:
Is it possible that route would be empty string? In such case `??` would be more suitable